src/dbms/handler.hpp

```diff
@@ -18,21 +18,21 @@
 #include <unordered_map>
 
 #include "global.hpp"
+#include "utils/exceptions.hpp"
+#include "utils/gatekeeper.hpp"
 #include "utils/result.hpp"
-#include "utils/sync_ptr.hpp"
 
 namespace memgraph::dbms {
 
 /**
  * @brief Generic multi-database content handler.
  *
- * @tparam TContext
- * @tparam TConfig
+ * @tparam T
  */
-template <typename TContext, typename TConfig>
+template <typename T>
 class Handler {
```

Comment:
Why adding also this layer, wouldn't dbms_handler be enough? Or do you know that Handler will be used for something in the future

Reply:
Its generic mechanism, dbms_handler use handler. Maybe could be move to utils...maybe in another PR.